src/write_outputs/write_outputs.jl

```diff
@@ -179,6 +179,14 @@ function write_outputs(EP::Model, path::AbstractString, setup::Dict, inputs::Dic
 				dfResMar_slack = write_reserve_margin_slack(path, inputs, setup, EP)
 			end		  
 		end
+
+		if setup["Reserves"]==1 && has_duals(EP) == 1
```

Comment:
I realize this code is duplicated from elsewhere in the file, but `has_duals` returns a boolean, so it's a bit redundant to use an `== 1`. You can just do `if setup["Reserves"]==1 && has_duals(EP)`.

Reply:
Thanks @cfe316 . Addressed these in all the instances